src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -301,31 +301,74 @@ def read_md_problem(filepath):
     return defdict_to_dict(return_dict, {})
 
 
-def dict_to_md(md_dict, remove_keys=[None]):
+def _remove_l3_headers(text: str, remove: set[str]) -> str:
+    """Removes specific level 3+ headers from a markdown string
```

Comment:
Can you just add a note for why this is needed? I think it's because the public version shouldn't have some of the pl-related panels right?

Reply:
Actually since this is the only change, I'll just do it.

Reply:
```suggestion
    """Removes specific level 3+ headers from a markdown string; useful for removing sections like pl-answer-panel and pl-submission panel for the public version of the site.
```